keanu-project/src/main/java/io/improbable/keanu/vertices/ProposalVertex.java

```diff
@@ -0,0 +1,7 @@
+package io.improbable.keanu.vertices;
+
+import java.util.List;
+
+public interface ProposalVertex {
```

Comment:
The proposal might not be a vertex. It might be a distribution represented by a bayesian network OR just a function that represents a distribution OR a single vertex that represents a distribution. Considering this interface is for KLDivergence and within the context of KL is very commonly called Q, maybe we could call it QDistribution and keep it in the same package as the KLDivergence class?